docker/geoserver/Dockerfile

```diff
@@ -4,32 +4,33 @@ LABEL GeoNode Development Team
 #
 # Set GeoServer version and data directory
 #
-ENV GEOSERVER_VERSION=2.18.2
+ENV GEOSERVER_VERSION=2.19.x
 ENV GEOSERVER_DATA_DIR="/geoserver_data/data"
 
 #
 # Download and install GeoServer
 #
 RUN cd /usr/local/tomcat/webapps \
-    && wget --no-check-certificate --progress=bar:force:noscroll https://www.dropbox.com/s/xlich7pmneaupqp/geoserver-${GEOSERVER_VERSION}.war?dl=1 -O geoserver-${GEOSERVER_VERSION}.war \
-    && unzip -q geoserver-${GEOSERVER_VERSION}.war -d geoserver \
-    && rm geoserver-${GEOSERVER_VERSION}.war \
+    && wget --no-check-certificate --progress=bar:force:noscroll https://artifacts.geonode.org/geoserver/${GEOSERVER_VERSION}/geoserver.war -O geoserver.war \
+    && unzip -q geoserver.war -d geoserver \
+    && rm geoserver.war \
```

Comment:
We could probably avoid to delete the `war` here, since the name matches the `webapps` folder, right?

Reply:
but @afabiani not having the war makes easier adding libs for testing and, first of all actually, it lessens a lot the docker image weight

Reply:
ok